apps/theme/layouts/forms/DefaultFormLayout.tsx

```diff
@@ -73,17 +146,39 @@ const DefaultFormLayout: FormLayoutComponent = ({
                             ))}
                         </Row>
                     ))}
-
                     {termsOfServiceEnabled && <TermsOfServiceSection component={TermsOfService} />}
                     {reCaptchaEnabled && <ReCaptchaSection component={ReCaptcha} />}
-
-                    <SubmitButton
-                        onClick={submit}
-                        loading={loading}
-                        fullWidth={formData.settings.fullWidthSubmitButton}
-                    >
-                        {formData.settings.submitButtonLabel || "Submit"}
-                    </SubmitButton>
+                    {/*
+                        If the form has more than one step then the form will be recognized as a Multi Step Form,
+                        so it means that we need to render form step handlers to switch between steps.
+                    */}
+                    {isMultiStepForm && (
+                        <ButtonsWrapper>
+                            <ButtonDefault onClick={handlePrevStep} disabled={currentStep === 0}>
```

Comment:
As mentioned in prev comment, we should be using `SubmitButton` here that was already here.